javascript/javascript1/week1/house-price.js

```diff
@@ -0,0 +1,34 @@
+// peter house measures
+let wide = 8;
+let deep= 10;
+let high = 10;
+let gardenSizeInM2 = 100;
+let volumeInMeters = wide * deep * high
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let houseValue = 2500000
```

Comment:
The price and value are swapped around.

Price is the 2_500_000.
Value is the derived amount based on the calculation. 

Then afterwards we need to ask if we 'should buy the house'. This is the case if the value is smaller or equals to the price.